include/rcutils/types/rcutils_ret.h

```diff
@@ -23,7 +23,7 @@ extern "C"
 #endif
 
 /// The type that holds a return value for an rcutils operation.
-typedef int rcutils_ret_t;
+typedef int32_t rcutils_ret_t;
```

Comment:
@clalancette how strict is ROS about ABI changes?

Reply:
Very strict.  We shouldn't do this in Galactic.

For that matter, a change like this should be targeting the `rolling` branch to start with, and then we can consider backporting it.

Reply:
https://github.com/TakanoTaiga/rcutils/blob/4eb5040ec5ea999d4d48be5df13bc5f6930f1854/src/error_handling_helpers.h#L137-L142

The precision is specified to prevent overflow of the buffer variable.

Reply:
Fixed the precision of ints to prevent overflow of Buffer variables.
https://github.com/TakanoTaiga/rcutils/blob/4eb5040ec5ea999d4d48be5df13bc5f6930f1854/src/error_handling_helpers.h#L137-L142

Reply:
Should I change the branch to `Rolling`  and submit a new PR?

Reply:
> Should I change the branch to Rolling and submit a new PR?

You don't need to submit a new PR.  You can just click "Edit" to the right of the title, and it will allow you to change the target branch.  Be aware, though, that you'll have to rebase your changes on top of the rolling branch and then force push to remove all of the conflicts that will occur from that action.

Reply:
I changed the branch to `Rolling`.

Reply:
changed branch to `Rolling`